con-base/src/main/java/ba/codecentric/base/domain/Hotel.java

```diff
@@ -62,6 +57,14 @@
     @Size(max = 500, message = "Description is too long, 500 characters allowed")
     private String description;
 
+    @NotNull(message = "Please add Check-in time")
+    @Column(name = "check_in")
+    private Time checkIn;
```

Comment:
Ok. I am thinking about it. But wouldn't this be same if we save it as String in db. If getter returns string, we will not have any use of Time data type. We just have unneeded parsing in setters and getters. Or I am wrong?